### PR TITLE
docs: fix specs for `ValidatorRewardScalar` and remove unnecessary image

### PR DESCRIPTION
**Pull request by 0xHansLee**

### CODE REVIEW

**Review by Pangssu — APPROVED** [review]

**Review by seolaoh — APPROVED** [review]
LGTM

**Review by kangsorang — APPROVED** [review]
LGTm

File: specs/assets/vscode_debug.png
Comment (seolaoh) [inline specs/assets/vscode_debug.png:1]: This file is used in `devnet.md`.

Comment (0xHansLee) [reply specs/assets/vscode_debug.png:1]: Thank you. I missed it.

**Pull request merged**